install/src/main/java/org/jetbrains/bsp/bazel/install/Install.kt

```diff
@@ -0,0 +1,71 @@
+package org.jetbrains.bsp.bazel.install
+
+import ch.epfl.scala.bsp4j.BspConnectionDetails
+import io.vavr.control.Try
+import org.jetbrains.bsp.bazel.install.cli.CliOptions
+import org.jetbrains.bsp.bazel.install.cli.CliOptionsProvider
+import org.jetbrains.bsp.bazel.installationcontext.InstallationContext
+import org.jetbrains.bsp.bazel.installationcontext.InstallationContextConstructor
+import kotlin.system.exitProcess
+
+object Install {
+
+    @JvmStatic
+    fun main(args: Array<String>) {
+        val cliOptionsProvider = CliOptionsProvider(args)
+        // TODO .get() wont be needed later (https://youtrack.jetbrains.com/issue/BAZEL-23)
+        val cliOptions = cliOptionsProvider.getOptions().get()
+        if (cliOptions.projectViewCliOptions != null) {
+            val projectViewCliOptions = ProjectViewCLiOptionsProvider()
+            projectViewCliOptions.generateProjectViewAndSave(cliOptions)
+        } else {
+            val default = ProjectViewDefaultInstallerProvider()
+            default.parseProjectViewOrGenerateAndSave(cliOptions)
+        }
+        if (cliOptions.helpCliOptions.isHelpOptionUsed) {
+            cliOptions.helpCliOptions.printHelp()
+        } else {
+            createEnvironmentAndInstallBazelBspServer(cliOptions)
+                    .onSuccess { printInCaseOfSuccess(cliOptions) }
+                    .onFailure(::printFailureReasonAndExit1)
+        }
+    }
+    private fun createEnvironmentAndInstallBazelBspServer(cliOptions: CliOptions): Try<Void> =
+            constructInstallationContext(cliOptions)
+                    .flatMap { createBspConnectionDetails(it, cliOptions) }
+                    .flatMap { createEnvironment(it, cliOptions) }
+
+    private fun constructInstallationContext(cliOptions: CliOptions): Try<InstallationContext> {
+        val resourcesProjectViewFilePath = "/default-projectview.bazelproject"
+        val projectViewProvider = ProjectViewDefaultFromResourcesProvider(
+                cliOptions.projectViewFilePath, resourcesProjectViewFilePath)
+        val projectViewTry = projectViewProvider.create()
```

Comment:
well, at the end of the day we are using only thins mechanism - this should be removed, and this method should get project view already provided earlier